pkgs/development/python-modules/hydrogram/default.nix

```
@@ -0,0 +1,52 @@
+{ lib
+, aiosqlite
+, buildPythonPackage
+, fetchFromGitHub
+, hatchling
+, pyaes
+, pysocks
+, pytest-asyncio
+, pytestCheckHook
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "hydrogram";
+  version = "0.1.4";
+  pyproject = true;
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "hydrogram";
+    repo = "hydrogram";
+    rev = "v${version}";
```

Helps to avoid branch/tag confusion.

```suggestion
    rev = "refs/tags/v${version}";
```